tests/test_keyboard_listeners.rs

```diff
@@ -0,0 +1,55 @@
+#[cfg(test)]
+mod tests {
+    use clipbond::listener::KeyboardListener;
+    use clipbond::listener::KeyboardListenerProvider;
+    use enigo::*;
+
+    static mut INDICATOR: bool = false;
+
+    fn callback() {
+        unsafe {
+            INDICATOR = !INDICATOR;
+        }
+        println!("callback was called");
+    }
+
+    fn press_key(modifier: Key) {
```

Comment:
probably press_c_key or something (again not enough arguments to make this function "press any key" generic